packages/common/src/core/types/IHubEditableContent.ts

```diff
@@ -19,6 +20,16 @@ export interface IHubEditableContent
    * capability enabled. This is a pre-requisite for Hosted Downloads to work.
    */
   serverExtractCapability?: boolean;
+  /**
+   * If the item represents a hosted feature service with "Extract enabled", shows the formats that
+   * can be extracted from the service via the "createReplica" operation.
+   */
+  serverExtractFormats?: string[];
+  /**
+   * links to additional resources specified in the formal item metadata
```

Comment:
if this comes from formal item metadata, but needs to be exposed at the top level, then let's just make a getter on the class for `this.metadata?.path.to.additionalResources`

Reply:
Good idea! Unfortunately, there is some conversion that needs to be made so we can't use a single getter. See this compose helper [here](https://github.com/Esri/hub.js/blob/master/packages/common/src/content/_internal/internalContentUtils.ts#L466).